src/Account/components/AccountHeaderCard.tsx

```diff
@@ -4,16 +4,16 @@ import CardContent from "@material-ui/core/CardContent"
 import IconButton from "@material-ui/core/IconButton"
 import MenuIcon from "@material-ui/icons/Menu"
 import makeStyles from "@material-ui/core/styles/makeStyles"
-import { Account } from "../../context/accounts"
-import { SettingsContext } from "../../context/settings"
-import { useIsMobile, useRouter } from "../../hooks/userinterface"
-import { matchesRoute } from "../../lib/routes"
+import { Account } from "~App/contexts/accounts"
+import { SettingsContext } from "~App/contexts/settings"
+import { useIsMobile, useRouter } from "~Generic/hooks/userinterface"
+import { matchesRoute } from "~Generic/lib/routes"
 import * as routes from "../../routes"
-import { breakpoints } from "../../theme"
-import { AccountCreation } from "../AccountCreation/types"
-import { Box } from "../Layout/Box"
-import withFallback from "../Lazy/withFallback"
-import ViewLoading from "../ViewLoading"
+import { breakpoints } from "~App/theme"
+import { AccountCreation } from "~AccountCreation/components/types"
```

Comment:
Maybe we should put the `types.ts` file inside `src/AccountCreation/` rather than `src/AccountCreation/components/`?

(I'd rather have commented on the actual `src/AccountCreation/components/types.ts` file but couldn't because it was only renamed and not changed.)

Reply:
Good point!